README.md

```diff
@@ -108,12 +120,6 @@ Check [examples](examples) and [interop](interop)(Interoperability Test) for som
 
 2. [Benchmark](benchmark) followed by official spec
 
-## Sponsors
```

Comment:
Is this sponsorship still going on? If not, I think it makes more sense to remove that section.

Reply:
I'm not sure. I think @tony612 is the best person to answer this

Reply:
It's okay to remove this.